mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -1606,10 +1607,11 @@ func isEqualProviderAutoScalingMaxInstanceSize(k, old, newStr string, d *schema.
 
 func clusterAdvancedConfigurationSchema() *schema.Schema {
 	return &schema.Schema{
-		Type:     schema.TypeList,
-		Optional: true,
-		Computed: true,
-		MaxItems: 1,
+		Type:       schema.TypeList,
+		Optional:   true,
+		Computed:   true,
+		ConfigMode: schema.SchemaConfigModeAttr,
```

Comment:
very interesting change, nice finding!
Did you notice any change in the structure of the terraform state when having this change from blocks to attributes? To make sure this change is not breaking, would it be worth adding a migration test case that creates the terraform state with previous version and reads it with the latest one?

Reply:
good point, let me work on that - thanks for the feedback

Reply:
nice, isn't this happening in other places?

Reply:
> this happening

what is "this"?

Reply:
other nested attributes that are computed or that might cause similar issues

Reply:
oh I get it: yes, it might be possible there are other nested blocks that have "computed: true". It might be worth it a follow-up action once we see the success of this.

Reply:
@AgustinBettati my newly created test is passing `--- PASS: TestAccClusterRSCluster_withDefaultBiConnectorAndAdvancedConfiguration_maintainsBackwardCompatibility (1033.39s)`

Now just waiting for the overall `cluster` test to succeed.